misk/src/main/kotlin/misk/exceptions/Exceptions.kt

```diff
@@ -0,0 +1,46 @@
+package misk.exceptions
+
+/** Common status codes for actions */
+enum class StatusCode(val code: Int) {
```

Comment:
Surely these must exist somewhere already? Guava?

Reply:
Guava doesn't have an HTTP stuff. They exist in Jersey, might existing in google's HTTP client.